front/src/components/NothingHere.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <q-page class="column justify-center">
+    <div class="text-center ">
+      <img style="width: 150px" :src="image || $t('defaultImageForNothingHereCmp')" />
```

Comment:
Please make sure to not use style=""; please use Quasar classes and layout function mentioned in task to set size.

Reply:
Sorry, I forgot to remove the style attribute. I've removed it now. Quasar doesn't have classes for fixed width like tailwind. I added a class in cmp scoped style.